buddyboss/buddyboss-platform
# PROD-739 #2947 - Fix: Profile edit > Field set issue in saving

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2947 .

### How to test the changes in this Pull Request:

1. Setup fieldset with repeater and make the fields inside it required.
2. Go to User profile > edit.
3. Try adding blank fieldsets.
4. Change some of the fields and delete empty fieldsets and save them.
5. Can see deleted fieldset will delete from the user profile.

### Proof Screenshots or Video
https://vimeo.com/624258970/c200ab761a


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
